src/index.ts

```diff
@@ -67,6 +88,7 @@ app.use(
         },
     }),
 );
+app.use(SSXExpressMiddleware(ssx));
```

Comment:
Adding the `SSXExpressMiddleware` gives you [multiple endpoints and authentication middleware](https://docs.ssx.id/ssx-how-it-works#ssx-server) for all requests